MainModule/Client/Core/Remote.lua

```diff
@@ -505,6 +509,29 @@ return function(Vargs, GetEnv)
 			local ms = ((math.floor((t2-t)*mult+0.5)/mult)*1000)
 			return ms
 		end;
+				
+		FPS = function()
+			local FPS = Remote.Get("FPS")
+			if not FPS then return false end			
+			local startTime = os.clock() 
+			local X = 1 
+			local FPS_Counter = 0
+			
+			local RenderStepped = service.RunService.RenderStepped
+			local Heartbeat = service.RunService.Heartbeat
+
+			local function GetFPS()
+				FPS_Counter+=1
+				if (os.clock() - startTime) >= X  then 
+					local fps = math.floor(FPS_Counter / (os.clock() - startTime))
+					return fps
+					FPS_Counter = 0
```

Comment:
Please fix.

Reply:
no. you fix it if your code is so good

Reply:
it's your PR, not mine; so you fix it if you are so convinced that your contribution is a good idea in the first place :p

Reply:
> nvm I'll fix it and get ez co-author :troll:

dude just close the PR and merge it yourself ezzzzzzz